internal/util/util.go

```diff
@@ -353,3 +354,15 @@ func CallStack() string {
 
 	return string(stack)
 }
+
+// IsVersionSatisfied check if available version satisfies required version
+func IsVersionSatisfied(required, available string) bool {
+	v1, _ := version.NewVersion(required)
+	v2, _ := version.NewVersion(available)
```

Comment:
are we discarding the error here?

Reply:
Yeah, I don't know how useful it would be to handle them. Not seen anything in examples handling them gracefully:
https://github.com/hashicorp/go-version#version-parsing-and-comparison

I can add one if you want.

Reply:
done will respin